inference-engine/src/mkldnn_plugin/nodes/common/cpu_convert.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "cpu_convert.h"
+#include "cpu_memcpy.h"
+#include <type_traits>
+#include <ie_parallel.hpp>
+
+using namespace InferenceEngine;
+
+template<typename srcType, typename dstType>
+void convert(void *srcPtr, void *dstPtr, const size_t &size) {
+    if (std::is_same<srcType, dstType>::value)
+        cpu_memcpy(dstPtr, srcPtr, size*sizeof(dstType));
```

Comment:
I think rest part of the function should be inside `else` branch otherwise you perform memory copy twice.

Reply:
thanks, it's my omission